.github/workflows/linux.yml

```diff
@@ -1350,6 +1350,89 @@ jobs:
             ${{ env.INSTALL_TEST_DIR }}/TEST*.html
           if-no-files-found: 'error'
 
+  TensorFlow_Hub_Performance_Models_Tests:
+    name: TensorFlow Hub Performance Models tests
+    needs: Build
+    defaults:
+      run:
+        shell: bash
+    runs-on: ${{ github.event_name == 'schedule' && 'ubuntu-20.04-16-cores' || 'ubuntu-20.04-8-cores'}}
+    timeout-minutes: ${{ github.event_name == 'schedule' && 400 || 5 }}
+    env:
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
```

Comment:
let's add `if: ` condition to run TF perf tests only when TF is changed

Example https://github.com/openvinotoolkit/openvino/blob/6c09fedc775d7d6edf1681a0f94ee6794251351d/.github/workflows/linux.yml#L1285

Reply:
Discussed it offline with @itikhono and @rkazants. Anyway we need to run tests. It doesn't matter changed TF or not

Reply:
TF label means that any other components are changes which affects TF somehow.
So, the label is still required, otherwise changes in GPU will trigger these tests, while we don't need it